src/platform/site-wide/banners/index.cypress.spec.js

```diff
@@ -0,0 +1,26 @@
+import Timeouts from 'platform/testing/e2e/timeouts';
+
+const SELECTORS = {
+  HOMEPAGE_BANNER: '[data-e2e-id="homepage-banner"]',
+  MAINTENANCE_BANNER_DOWNTIME: '[data-e2e-id="maintenance-banner-downtime"]',
+  MAINTENANCE_BANNER_PRE_DOWNTIME:
+    '[data-e2e-id="maintenance-banner-pre-downtime"]',
+};
+
+describe('Banner A11y Check', () => {
+  // Test skipped as it was disabled in Nightwatch
```

Comment:
Is this even a valid test? I checked the home page, and those selectors do not work.

Are we planning to fix the test?

Reply:
I'm honestly not sure what the plans are for the test. It was being disabled in Nightwatch (assuming because it was either outdated or non-functional) however it was not ever removed or fixed, so it's been copied over in it's current state just to ensure we do not lose it in the event it was a WIP when Nightwatch is removed.  Once the conversion has been made, we'll be reevaluating the suite as a whole and communicating with any necessary parties about tests that may no longer be necessary (as in something like this, that very well may be covered by other accessibility checks) or tests that are not currently being utilized and what their future plans are.

Reply:
I understand that the intention here is a "lift and shift", but I see no reason to port a test that adds no value.

Reply:
Totally agree with the sentiment here.  My only concern was with the permanent removal of nightwatch that someone may lose something they intended to come back to.  Ran this specific spec by my team and they agreed that we can probably drop this as well, so removing this one.